src/components/Records/Search/Input/FilterButton.vue

```diff
@@ -53,13 +54,13 @@
             const fieldValue = _module.currentParameter[this.item.filterName];
             const currentValue = _module.item.value;
             const title = _module.item.title.toLowerCase();
-            _module.checkCurrentParameters(title, fieldValue, currentValue);
+              _module.checkCurrentParameters(title, fieldValue, currentValue);
           });
         },
         methods: {
             checkCurrentParameters: function(title, fieldValue, currentValue) {
               if (fieldValue === null) {
-                this.item.active = title === 'all';
+                this.item.active = title === 'all' || title === 'match all term';
```

Comment:
this line doesn't make sense. It's supposed to be || title === "match all terms"